package.json

```diff
@@ -22,17 +22,17 @@
     "wdio-standard": "wdio run ./tests/wdio-standard.conf.js"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "^10.0.1",
-    "@angular-devkit/build-angular": "^0.1000.8",
-    "@angular-eslint/builder": "^0.6.0-beta.0",
-    "@angular-eslint/eslint-plugin": "^0.6.0-beta.0",
-    "@angular-eslint/eslint-plugin-template": "^0.6.0-beta.0",
-    "@angular-eslint/schematics": "^0.6.0-beta.0",
-    "@angular-eslint/template-parser": "^0.6.0-beta.0",
-    "@angular/cli": "^10.0.8",
-    "@angular/compiler": "^10.0.14",
-    "@angular/compiler-cli": "^10.0.14",
-    "@angular/localize": "^10.0.14",
+    "@angular-builders/custom-webpack": "^12.1.0",
```

Comment:
Dang it. We can't joke about "[goes to 11](https://youtu.be/uMSV4OteqBE?t=77)" if we skip that and go to 12!

Reply:
Haha there was a small window to do that during the first few commits, you had to be fast!